packages/build/src/plugins/manifest/path.js

```diff
@@ -7,7 +7,13 @@ export const getManifestPath = async function ({ pluginDir, packageDir, packageN
   const dirs = [pluginDir, packageDir]
     .filter(Boolean)
     .flatMap((dir) => MANIFEST_FILENAMES.map((filename) => `${dir}/${filename}`))
-  const manifestPath = await locatePath(dirs)
+  let manifestPath
+  for (const dir of dirs) {
```

Comment:
nit - we could potentially leverage `find` to follow the same functional style that is already taking place in this method